cloudvolume/cloudvolume.py

```diff
@@ -143,11 +136,22 @@ def create_new_info(cls, num_channels, layer_type, data_type, encoding, resoluti
   @classmethod
   def extract_path(cls, cloudpath):
     """cloudpath: e.g. gs://neuroglancer/DATASET/LAYER/info or s3://..."""
-    match = re.match(r'^(gs|file|s3|boss)://(/?[\d\w_\.\-]+)/([\d\w_\.\-]+)/([\d\w_\.\-]+)/?', cloudpath)
-    return ExtractedPath(*match.groups())
+    protocol_re = r'^(gs|file|s3|boss)://'
+    tail_re = r'(/?[\d\w_\.\-]+)/([\d\w_\.\-]+)/([\d\w_\.\-]+)(?:/info|/provenance|/)?$'
```

Comment:
Accepting an info there is sort of problematic, what if my layer name is `info`?
Interpreting "common" mistake vs making the definition simpler.
I would maybe check and return a message like:
"Is your layer actually call info? otherwise remove info from here"

Reply:
We discussed and think that:
1) Neuroglancer doesn't support trailing info/provenance so we'd be training people in two different ways. Furthermore, even if we change our version of neuroglancer, there are many that won't be changed.
2) We can identify problems and issue corrective advice if the info fetch fails and the layer is named info.